Code/autopkg

```diff
@@ -735,9 +747,8 @@ Example: '%s repo-add recipes'
 ..adds the autopkg/recipes repo from GitHub."""
         % ("%prog", verb, "%prog")
     )
-
     # Parse arguments
-    arguments = parser.parse_args(argv[2:])[1]
+    arguments = common_parse(parser, argv[2:])[1]
```

Comment:
The old behavior was to set `arguments` to the parsed arguments, but the new code actually sets `arguments` to a tuple containing `(options, arguments)`. I suspect this is why I can't run `autopkg run` or `autopkg make-override` successfully on this branch.